blocks/community-cards/community-cards.css

```diff
@@ -0,0 +1,162 @@
+.community-cards .cc-container {
+  gap: 2rem;
+  color: var(--text-color);
+  display: flex;
+  overflow-x: scroll;
+  font-size: var(--body-font-size-xxs);
+}
+
+.community-cards .cc-container::-webkit-scrollbar {
+  display: none;
+}
+
+.community-cards .cc-container .card {
+  cursor: default;
+  background: var(--body-text-color) 0 0 no-repeat padding-box;
+  box-shadow: 0 3px 6px #000029;
+  border-radius: 16px;
+  opacity: 1;
+  flex: 1;
+  min-width: 240px;
+  overflow: none;
+  padding: 1.5rem 0;
+  position: relative;
+}
+
+.community-cards .cc-container .card .card-header{
+  display: flex;
+  padding-left: 1rem;
+  width: 65%;
+}
+
+.community-cards .cc-container .card .card-header h3{
+  margin: 0;
+  flex: 1;
+  line-height: 1.25;
+  font-weight: 700;
+  font-size: var(--body-font-size-xs);
+}
+
+.community-cards .cc-container .card .card-category{
+  font-weight: 650;
+  line-height: 1.25;
+  padding: 8px 4px;
+  background-color: var(--very-light-gray);
+  border-radius: 4px 0 0 4px;
+  position: absolute;
+  right: 0;
+  top: 13px;
+}
+
+.community-cards .cc-container .card .card-description{
+  padding: 0 1rem;
+  margin: 0.5rem 0 1.188rem;
+}
+
+.community-cards .cc-container .card .card-meta{
+  padding: 0 1rem;
+  margin: 0 0 1rem;
+}
+
+.community-cards .cc-container .card .card-meta .card-meta-first-section{
+  display: flex;
+}
+
+.community-cards .cc-container .card .card-meta .card-meta-first-section .card-meta-items{
+  display: flex;
+  align-items: center;
+}
+
+.community-cards .cc-container .card .card-meta .card-meta-first-section .card-meta-items .meta-separator{
+  display: none;
+}
+
+.community-cards .cc-container .card .card-meta .card-meta-first-section .card-meta-items.owner{
+  justify-content: right;
+  flex: 1;
+}
+
+.community-cards .cc-container .card .card-meta .card-meta-first-section .card-meta-items.owner img{
+  margin: 0 5px;
+  width:24px;
+  height:24px;
+}
+
+.community-cards .cc-container .card .card-meta .card-meta-first-section .card-meta-items.visibility .icon-globe{
```

Comment:
Could we make this selector as shorter. It's more than 5 level of depth. @akki761